### PR TITLE
Enhancement: Sourcemap `mapping` field deprecation

### PR DESCRIPTION
**Pull request by ahangsu**
Implements https://github.com/algorand/algorand-sdk-testing/issues/202 here

### CODE REVIEW

File: .test-env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="revert-sandbox"
```
Comment (ahangsu) [inline .test-env:3]:
```suggestion
SDK_TESTING_BRANCH="master"
```

**Review by algochoi — APPROVED** [review]
I think this looks good - since the test itself is not a breaking change, we could also target `master` for this PR right?

**Comment by ahangsu** [comment]
I think for now all the testing branches are targeting master?

**Comment by algochoi** [comment]
Oh I mean this PR itself (currently targeting [feature/v2.0.0-break](https://github.com/algorand/java-algorand-sdk/tree/feature/v2.0.0-break))

**Comment by ahangsu** [comment]
I think it is part of https://github.com/algorand/java-algorand-sdk/issues/389, so I'm pointing to feature branch for now.

**Comment by ahangsu** [comment]
After more thoughts, I think you are right. If this PR does not go to master, then the current change of testcase might fail.

**Pull request merged**